caffeinated/admin/extend/about/templates/ee4-overview.template.php

```diff
@@ -173,18 +172,25 @@
                     '</a>'
                 ); ?></p>
         </div>
-        <div><img src="<?php echo EE_GLOBAL_ASSETS_URL; ?>images/screenshots/single-event-page.jpg"></div>
+        <div class="columns"><img src="<?php echo EE_GLOBAL_ASSETS_URL; ?>images/screenshots/single-event-page.jpg"></div>
     </div>
 
-
-</div>
 <hr>
 
-<h3><?php esc_html_e('Pick a theme, any theme', 'event_espresso'); ?></h3>
-<p><?php
-    esc_html_e(
-        'We’ve made it super easy to integrate Event Espresso with almost any properly coded WordPress theme, including many of the thousands of themes available on WordPress.org. The image below shows the same Event Espresso ticketing page across three diffrent WordPress themes.',
-        'event_espresso'
-    ); ?></p>
-<p><img class="about-overview-img" src="<?php echo EE_GLOBAL_ASSETS_URL; ?>images/screenshots/multiple-themes.jpg"/></p>
-<hr>
+    <h2 class="about-headline-callout">
+        <?php esc_html_e('Pick a theme, any theme'); ?>
```

Comment:
I'm not sure if the i18n domain argument was deliberately left out here, but phpcs is complaining about it.  So if intentional (because the string is translated for WordPress core, then there should be an phpcs ignore comment added here.